tests/magma_linalg_test.py

```diff
@@ -114,5 +114,40 @@ def testEigMagmaConfig(self):
       hlo = jax.jit(partial(lax_linalg.eig, use_magma=True)).lower(a).as_text()
       self.assertIn('magma = "on"', hlo)
 
+  @jtu.sample_product(
+    shape=[(3, 4), (3, 3), (4, 3), (4, 3)],
+    dtype=float_types + complex_types,
+  )
+  @jtu.run_on_devices("gpu")
+  def testPivotedQrFactorization(self, shape, dtype):
+    if jtu.jaxlib_version() <= (0, 4, 38):
```

Comment:
This probably needs to be updated to `(0, 5, 0)` now that that's the most recent release.